src/tiles/FeedTile.vue

```diff
@@ -1,15 +1,71 @@
 <template>
   <article class="tile feed">
-        <h2 class="feed--title">Ogłoszenia</h2>
-        <div class="feed--content">
-            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed dignissim, est quis molestie faucibus, ligula urna mollis dui, in gravida nibh libero id risus. Donec eget nibh eget sem suscipit sodales. Nam interdum velit dolor, eget placerat turpis tincidunt et. Fusce a augue nunc. Aenean volutpat ut risus eget vestibulum. In luctus odio in libero congue ullamcorper quis vitae enim. Duis tempor purus id tellus condimentum pretium. Aliquam in ligula aliquet, fermentum lectus sit amet, scelerisque felis. Curabitur at tempor velit.
-            Sed vestibulum suscipit efficitur. Quisque ac pretium urna, at ornare lorem. Proin metus felis, tristique a egestas et, aliquet at mauris. Morbi nec pellentesque ipsum. Nam consectetur justo id sollicitudin bibendum. Suspendisse potenti. Morbi mollis pharetra dolor, a lobortis sem. Nam quis varius neque. Proin imperdiet dignissim dui, a suscipit dui lobortis id. Integer iaculis id felis nec semper. Donec nisl magna, condimentum sed nisi in, dictum cursus ligula. Proin efficitur risus semper commodo sollicitudin. Phasellus ullamcorper mauris nec quam tempus sollicitudin.
+        <h2 class="feed--advert">Ogłoszenia</h2>
+        <div class="feed--marquee" id="marquee">
+            <div class="feed--scroll" id="vmarquee">
+                <div v-for="feed in feeds">
+                    <div class="feed--title">{{ feed.title }}</div>
+                    <div class="feed--content">{{ feed.content }}</div>
+                    <div class="feed--author">{{ feed.author.username }}</div>
+                    <div class="feed--publishedAt">{{ feed.publishedAt.toLocaleString() }}</div>
+                </div>
+            </div>
         </div>
     </article>
 </template>
 
 <script>
+    import * as marquee from './../marquee'
+    import fetch from 'fetch-retry'
+    import { API_URL } from '../constants'
+
     export default {
+        data() {
+            return {
+                feeds:[{
+                        "title": "",
+                        "content": "",
+                        "author": {
+                            "username": ""
+                        },
+                        "publishedAt": ""
+                    }
+                ]
+            }
+        },
+
+        mounted() {
+            this.updateNews()
+
+            setInterval(() => {
+                this.updateNews()
+            }, 1000 * 60 * 5); // Update news every 5 minutes
+            marquee.Init('marquee','vmarquee');
```

Comment:
Blank line before this line.

Reply:
And IMO you should not pass this second parameter, but get child of first parameter